src/main.cpp

```diff
@@ -1220,10 +1220,10 @@ bool AcceptToMemoryPoolWorker(CTxMemPool& pool, CValidationState &state, const C
 }
 
 bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransaction &tx, bool fLimitFree,
-                        bool* pfMissingInputs, bool fOverrideMempoolLimit, bool fRejectAbsurdFee)
+                        bool* pfMissingInputs, bool fOverrideMempoolLimit, const CAmount nAbsurdFee)
```

Comment:
@jtimon I never said it was worse, it is just a convention we use nowhere else, whereas `const CAmount& someAmount` is a common convention (according to `git grep`).

There's no deep reason not to make pass-by-value arguments to a function const. On the other hand, it is *only* a programmer convention for marking a value as "Do not change this!". The compiler will still happily push a copy on the stack.

So usually that means someone made a typo. If this is intentional I'm fine with it, I didn't intend for this to become a whole argument.